components/client/src/main/java/com/hotels/styx/client/connectionpool/SimpleConnectionPool.java

```diff
@@ -393,8 +414,14 @@ public Factory recordStats(boolean recordStats) {
             return this;
         }
 
+        public Factory connectionDecorator(ConnectionDecorator connectionDecorator) {
+            this.connectionDecorator = connectionDecorator;
+            return this;
+        }
```

Comment:
No longer used?

Also my editor says that `recordStats` method above is no longer used. Please review if they can be removed.